git-check-attr: add page (#5098)
tldr-pages/tldr: pages/common/git-check-attr.md

# git check-attr

> For every pathname, list if each attribute is unspecified, set, or unset as a gitattribute on that pathname.
> More information: <https://git-scm.com/docs/git-check-attr>.

- Check the values of all attributes on a file:

> `git check-attr -a {{path/to/file}}`

Reviewer: ```suggestion
`git check-attr --all {{path/to/file}}`
```

- Check the value of a specific attribute on a file:

`git check-attr {{attribute}} {{path/to/file}}`

- Check the value of a specific attribute on files:

> `git check-attr -a {{path/to/file1}} {{path/to/file2}}`

Reviewer: ```suggestion
`git check-attr --all {{path/to/file1}} {{path/to/file2}}`
```

> - Check the value of a specific attribute on a file:

Reviewer 2: Perhaps this will read better?
```suggestion
- Check the value of a specific attribute on one or more files:
```
